tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -668,13 +658,16 @@ public function resetOldLdapConfig() {
 	 * @throws Exception
 	 */
 	public function afterScenario() {
-		if ($this->getLdapTestStatus()) {
+		if ($this->isTestingWithLdap()) {
 			$this->deleteLdapUsersAndGroups();
 			$this->resetOldLdapConfig();
 		}
 	}
 
 	/**
+	 * This function will allow us to send user creation requests in parallel.
+	 * This will be faster in comparision to waiting for each request to complete before sending another request.
```

Comment:
```suggestion
	 * This will be faster in comparison to waiting for each request to complete before sending another request.
```
only if we need to make other changes

Reply:
done!